src/Fmt_odoc.ml

```diff
@@ -90,11 +89,11 @@ and fmt_text_elt = function
       hvbox 0 (wrap "{!modules:" "}" (list l "@," str))
   | Special_ref SRK_index_list -> str "{!indexlist}"
   | Target (s, l) ->
+      let target = Option.value_map s ~default:"" ~f:(fun s -> s ^ ":") in
+      let latex = String.is_empty target || String.equal target "latex:" in
```

Comment:
Is it clear that latex is special here, or should all `Target`s be left unescaped?

Reply:
I didn't find any info suggesting that latex is a special case, so I don't escape any target anymore